phpBB/phpbb/db/migration/data/v400/remove_notify_type.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v400;
+
+use phpbb\db\migration\migration;
+
+class remove_notify_type extends migration
+{
+	public static function depends_on(): array
+	{
+		return [
+			'\phpbb\db\migration\data\v400\remove_jabber',
+			'\phpbb\db\migration\data\v400\dev',
+			'\phpbb\db\migration\data\v30x\release_3_0_0',
+		];
+	}
+
+	public function update_schema(): array
+	{
+		return [
+			'drop_columns' => [
+				$this->table_prefix . 'users' => [
+					'user_notify_type',
+				],
+			]
+		];
+	}
+
+	public function revert_schema(): array
+	{
+		return [
+			'add_columns' => [
+				$this->table_prefix . 'users' => [
+					'user_notify_type' => ['TINT:4', 0],
+				],
+			]
+		];
+	}
+
+	public function update_data()
+	{
+		return [
+			['custom', [[$this, 'remove_jabber_user_notifications']]],
+		];
+	}
+
+	protected function remove_jabber_user_notifications(): void
+	{
+		$sql = 'DELETE FROM ' . $this->tables['user_notifications'] . "
+			WHERE method = 'notification.method.jabber'";
+		$this->db->sql_query($sql);
```

Comment:
`notification.method.jabber` was previously replaced with `notification.method.email` in `phpbb/db/migration/data/v400/remove_jabber.php`, so it looks like there will be nothing to delete here.